src/showcase/src/flows.ts

```diff
@@ -26,7 +31,37 @@ export const flowsPage = () => {
   render(pageContent, container);
 };
 
-const registerFlowOpts: RegisterFlowOpts<null> = {
+const mockDelegationIdentity = {} as DelegationIdentity;
+const mockActor = {
+  identity_info: () => {
+    return {
+      Ok: {
+        authn_methods: [],
+        metadata: [],
+        authn_method_registration: [],
+      },
+    };
+  },
+} as unknown as ActorSubclass<_SERVICE>;
+
+class MockAuthenticatedConnection extends AuthenticatedConnection {
```

Comment:
Not in this PR: We should extract an interface rather than extending the real implementation...

Reply:
Yes, that sounds better.